### PR TITLE
🛑 Stop autoplay when commenting

### LINKED ISSUE #2958: Pause autoplay when user is typing a comment
Imagine you just watched a video and are commenting about it. Atlas automatically skips to the next video and looses your writing. Just happened to me. User thinks this is a bad app and will be demotivated to interact again.

### PR DESCRIPTION
**Pull request by WRadoslaw**
fixes #2958

### CODE REVIEW

**Review by kdembler — APPROVED** [review]
LGTM

**Comment by kdembler** [comment]
@WRadoslaw could you add an entry in `CHANGELOG.md` before we merge?

**Comment by WRadoslaw** [comment]
> @WRadoslaw could you add an entry in `CHANGELOG.md` before we merge?

Sure

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
+- Autoplay stop when user starts commenting
```
Comment (kdembler) [inline CHANGELOG.md]:
```suggestion
### Added

- Stop video autoplay if user starts commenting
```

**Review by kdembler — APPROVED** [review]

**Pull request merged**